graphite/graphite.go

```diff
@@ -3,18 +3,23 @@ package graphite // import "bosun.org/graphite"
 
 import (
 	"encoding/json"
-	"errors"
 	"fmt"
+	"io/ioutil"
 	"net/http"
 	"net/url"
+	"strings"
 	"time"
 )
 
+const ParseErrFmt = "graphite ParseError (%s): %s"
```

Comment:
really? to me this makes sense to keep it in the graphite package. semantically (it defines stuff that is specific to graphite) and also because we define the other error here.

but if you feel strongly about this, i guess i'll move it...